src/contracts/middleware/IndexRegistry.sol

```diff
@@ -235,19 +240,19 @@ contract IndexRegistry is IIndexRegistry {
 
     /// @notice Returns the index of the `operatorId` at the given `blockNumber` for the given `quorumNumber`, or max uint32 if the operator is not active in the quorum
```

Comment:
`or max uint32 if the operator is not active in the quorum` => `or OPERATOR_DEREGISTERED_INDEX if the operator was not registered for the `quorumNumber` at `blockNumber`.

Reply:
40533cf7